inspectit-oce-agent/src/system-test/resources/config/UserInstrumentationWithMetricsTest.yml

```diff
@@ -0,0 +1,51 @@
+inspectit:
+  metrics:
+    definitions:
+      '[my/invocation]':
+        type: LONG
+        unit: invocations
+        views:
+          '[my/invocation/count]':
+            aggregation: count
+            with-common-tags: false
+            tags:
+              method_name: true
+              user_tag: true
+          '[my/invocation/sum]':
+            aggregation: sum
+            with-common-tags: false
+            tags:
+              method_name: true
+
+  instrumentation:
+    scopes:
+      UserInstrumentationWithMetricsTest-invocationCount:
+        type: &UserInstrumentationWithMetricsTest
```

Comment:
We have to be carefull that we dont make it too complex all the stuff. This references are nice but the configuration is already complex enough :)